plugins/aws/sqs/sqsHasTags.spec.js

```diff
@@ -0,0 +1,92 @@
+var expect = require('chai').expect;
+var sqsHasTags = require('./sqsHasTags');
+
+const createCache = (listQueues, rgData) => {
+    return {
+        sqs: {
+            listQueues: {
+                'us-east-1': {
+                    err: null,
+                    data: listQueues
+                }
+            },
+        },
+        resourcegroupstaggingapi: {
+            getResources: {
+                'us-east-1':{
+                    err: null,
+                    data: rgData
+                }
+            }
+        }
+    };
+};
+
+describe('SQS Has Tags', function () {
+    describe('run', function () {
+        // it('should PASS if SQS queues have tags', function (done) {
+        //     const cache = createCache(['https://sqs.us-east-1.amazonaws.com/111122223333/test-queue-1'], [
+        //         {
+        //             "ResourceARN": "arn:aws:sqs:us-east-1:111122223333:test-queue-1",
+        //             "Tags": [{ key: 'name', value: 'test-queue-1' }],
+        //         }
+        //     ]);
+        //     sqsHasTags.run(cache, {}, (err, results) => {
+        //         expect(results.length).to.equal(1);
+        //         expect(results[0].status).to.equal(0);
+        //         expect(results[0].region).to.equal('us-east-1');
```

Comment:
update the message in spec file and pass check if sqs queue has tags is commented